doc/user/content/transform-data/patterns/durable-subscriptions.md

```diff
@@ -1,42 +1,45 @@
+title: "Durable subscription (Retain History)"
+description: "How to configure history retention to set up durable subscriptions in Materialize"
 menu:
   main:
     parent: 'sql-patterns'
-    name: "Time travel queries"
+aliases:
+  - /transform-data/patterns/time-travel-queries/
+  - /transform-data/patterns/time-travel-queries/#history-retention-period
 ---
 
-{{< private-preview />}}
+You can configure an object's [history retention
+period](#history-retention-period) for durable subscriptions.
 
-By default, all user-defined sources, tables, materialized views, and indexes
-keep track of the most recent version of their underlying data. Some use cases
-require accessing past versions of this data at specific points in time
-(i.e. time traveling). To allow time travel queries, you can configure these
-objects to **retain history**.
+{{< private-preview />}}
 
-When set to a value different than the default (i.e. no history retention), the
-history retention period of an object determines the duration for which
-historical versions of its underlying data are available for querying.
+## Durable subscriptions
 
-The common use cases for time traveling in Materialize are:
+Because [`SUBSCRIBE`](/sql/subscribe/) requests happen over the network, these
+connections might get disrupted for both expected and unexpected reasons. To
+avoid the need for re-snapshotting the data, you can:
 
-* Lossless, **durable subscriptions** to your changing results. See the
-  [durable subscriptions section below](#durable-subscriptions)
-  for details on how to create this type of subscription.
+- Adjust the [history retention period](#history-retention-period) for the
+  objects that a subscription depends on, and then
 
-* Accessing a past version of results at a specific point in time.
+- Use [`AS OF`](/sql/subscribe#as-of) to pick up where you left off on
+  connection drops.
 
 ## History retention period
 
-You can change the history retention period for [sources](/sql/create-source/),
-[tables](/sql/create-table/), [materialized views](/sql/create-materialized-view/),
-and [indexes](/sql/create-index/). To configure the history retention period for
-an object, use the `RETAIN HISTORY` option in the `CREATE` statement. This
-value can also be adjusted at any time using the object-specific `ALTER`
-statement.
+{{< private-preview />}}
 
-### Configuration
+The history retention period of an object ([sources](/sql/create-source/),
+[tables](/sql/create-table/), [materialized
+views](/sql/create-materialized-view/), and [indexes](/sql/create-index/))
+determines the duration for which historical versions of the object's underlying
+data are available for querying. By default, history retention period is unset
+for all user-defined sources, tables, materialized views, and indexes; that is,
+these objects keep track only of the most recent version of their underlying
+data. You can modify this retention period using the `RETAIN HISTORY` option.
+
+### Behavior
```

Comment:
```suggestion
### Semantics
```

Reply:
All of this should live somewhere else since it's reference documentation. Surfacing a comment I left in the original PR (#26623):

> Eventually, I think we'll want to extricate the SQL reference bits as a shortcode that can be applied to the low-level reference pages, and focus this page on just guidance on how to implement the time traveling patterns in practice.

Reply:
100% agreed.  Yeah ... I noticed that this page seemed more SQL reference than how to implement the durable subscription using a feature of our product.

Reply:
I meant this as a comment for a future pass, if you don't have the bandwidth right now. This is the current state, anyways, so it's not like it'll get worse. 🫠 Probably something we'll just forget to do if we don't do it now, is more the thing!

Reply:
Since this is the single use case for this option, I can implement the shortcode in the next patch.  That way, this page really can be about implementing durable subscription instead of trying to be about that + a reference section on the history retention feature.